### ISSUE TITLE
Activity?

### ISSUE THREAD

**Issue by natrius** [Al-th/Vocals#18]
Are there any updates or further development planned?

**Comment by ghost** [comment]
I'm also interested in getting this rebooted. If you get no response lets fork it!~

**Comment by Al-th** [comment]
@natrius  : No, there are no updates nor any further development planned yet on my side. 

@etisdew  : The code has always been open source. If you want to fork it, go for it, it will only be good for the community. If anything, I would be happy if you could reference Vocals.

**Comment by ghost** [comment]
Absolutely, great work man.

**Comment by ghost** [comment]
If it wouldnt be asking too much anytime you're not busy. I'd be very grateful for a bit more documentation, this isnt my native language and I'm not familiar with your workflow if there are any nuances before I start desecrating it with naive bumbling. I wouldn't demand it be in English but this is a great project that should live. I'd be happy to make sure it flowed in English and spend some time translating.

**Comment by ghost** [comment]
I'm using it with .ahk scripts. Could not be happier, could I get any suggestions on how to really get the most out of it?

**Comment by Al-th** [comment]
Hi,
Unfortunately no, I do not have time to document more. I don't get your problem though : all methods are quite small and written in English, aren't they ?

Besides, I don't know ahk scripts so I cannot answer your second question.

**Comment by ghost** [comment]
I am serious about this project and would like on the team. I will begin by developing documentation and a new React site. You deserve it. Je vous en remercie.

Are you considering either of the PRs though? They slightly break between builds and I'd like to foresee that.

**Comment by Al-th** [comment]
I am sincerely happy to see your motivation w.r.t this project. Nevertheless, as I said I absolutely do not have the possibility to invest time in this project right now. I'm finishing my Ph.D and that does not allow me to spend time on it neither to want to spend time on it when I have some free time.

As a result, while I absolutely will consider the PR's at some point, it is absolutely not my priority (and don't get me wrong, I absolutely love the fact that people proposed changes). The thing is : if I were to consider those PR's, I would also have to verify that they do not break the current software, which ... I do not have time to.

Moreover, this project is several years old right now. If you are really interested in the project, I would advise to adapt the project to modern computer OS, constraints and libraries by doing one of your own. That would allow the open-source community to have a software of their own that is "up-to-date". And I feel that you seem to be the right guy for the job.
